website/templates/project/addon/node_settings_default.mako

```diff
@@ -34,7 +34,11 @@
                 <p class="break-word">
                     <strong>Current Folder:</strong>
                     <a data-bind="ifnot: folderName() === '', attr.href: urls().files">
-                        {{folderName}}
+                        %if (addon_short_name == 'googledrive'):
+                            {{decodedFolderName}}
+                        %else:
+                            {{folderName}}
+                        %endif
```

Comment:
Would it be possible to make `folder_name` decoded instead of having a separate `decodedFolderName` just for google drive?